client/src/components/polls/PollImages.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
```

Comment:
We have a component with the same name at `src/pages/Dashboard/PollImages.js`, so we should consider re-naming this

Reply:
I didnt notice that. But you're right, if you have the time you could change this in another PR. Honestly there are many things that need to be better organized/named, but I don't think we should spend time on that.